core/src/components/range/range.md.scss

```diff
@@ -130,11 +157,38 @@
   }
 }
 
-.range-knob-pressed .range-pin {
+.range-knob-pressed .range-pin,
+.range-knob-handle.ion-focused .range-pin {
   @include transform(translate3d(0, -24px, 0), scale(1));
 }
 
-:host(:not(.range-has-pin)) .range-knob-pressed .range-knob {
+@media (any-hover: hover) {
+  .range-knob-handle:hover .range-knob:before {
+    transform: scale(2);
+    
+    opacity: 0.04;
```

Comment:
Thoughts on making the hover opacity the same as the focus opacity? I can't really see `0.04` 
![image](https://user-images.githubusercontent.com/2721089/189432787-666065e0-2200-4e2f-8d33-312bdd6d9858.png)

The main distinction between hover + focus would be the scale of the outline/glow.

Reply:
I'll reach out to Ben Monday on this. The opacity values selected were calculated by taking the color samples from the MD spec and applying an opacity that matched the new background color for each indicator state.

Even in the spec screenshot, it isn't shown on a white background, which makes me think the hover opacity almost being not visible could be deliberate.